skyportal/handlers/api/photometry.py

```diff
@@ -54,6 +56,80 @@
 
 MAX_NUMBER_ROWS = 10000
 
+cmap_ir = cm.get_cmap('autumn')
+
+
+def hex2rgb(hex):
+    return tuple(int(hex[i : i + 2], 16) for i in (0, 2, 4))
+
+
+def get_effective_wavelength(bandpass_name):
+    try:
+        bandpass = snc.get_bandpass(bandpass_name)
```

Comment:
Make an optional argument for radial dependent bandpass a la: sncosmo.get_bandpass('"ultrasat', 5.0)

Reply:
Ah some bands require a second argument that is numerical corresponding to position on the focal plane, like Ultrasat.

Reply:
Got it. So I can add the argument here without any problem, but I suppose we won't use it here for the plotting right?

Reply:
Yeah not yet, just noticed we will need it for certain bands